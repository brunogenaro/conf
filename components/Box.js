import React, { PureComponent } from 'react';

import Globals from '../utils/Globals';

export default class extends PureComponent {
  render() {
    const { children } = this.props;

    return (
      <div className='cont'>
        { children }

        <style jsx>{`
          .cont {
            border-top: 3px solid ${Globals.colors.primary};
            text-align: left;
            margin: 30px;
            text-transform: uppercase;
            @media(max-width: 720px) {
              margin: 5px;
            }
          }
        `}</style>
      </div>
    );
  }
}
