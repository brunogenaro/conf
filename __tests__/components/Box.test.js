import React from 'react';
import renderer from 'react-test-renderer'
import Box from '../../components/Box.js';

describe('With Snapshot Testing', () => {
  it('Box Snapshot Testing', () => {
    const component = renderer.create(<Box />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot()
  })
});