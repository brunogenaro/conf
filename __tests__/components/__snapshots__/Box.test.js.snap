// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Box Snapshot Testing 1`] = `
<div
  className="cont"
  data-jsx={3134665551}
/>
`;
